src/main/scala/beam/cosim/helics/BeamHelicsInterface.scala

```diff
@@ -0,0 +1,270 @@
+package beam.cosim.helics
+
+import beam.agentsim.scheduler.Trigger
+import com.github.beam.HelicsLoader
+import com.java.helics._
+import com.java.helics.helicsJNI._
+import com.typesafe.scalalogging.StrictLogging
+import spray.json.DefaultJsonProtocol.{listFormat, mapFormat, JsValueFormat, StringJsonFormat}
+import spray.json.{JsNumber, JsString, JsValue, _}
+
+object BeamHelicsInterface {
+  // Lazy makes sure that it is initialized only once
+  lazy val loadHelics: Unit = HelicsLoader.load()
```

Comment:
This is confusing from a naming convention. Admittedly it is hard to name - possibly `loadHelicsIfNotAlreadyLoaded`? I'm a fan of readability over anything else. Otherwise it does beg the question as to whether it should be called as many times as it is (more an architectural question).

Reply:
good point. name changed.